src/ng1/services/colorService/colorService.service.js

```diff
@@ -80,6 +85,20 @@ export default function $colorService() {
     }
   };
 
+  $colorService.resolve = function resolve(value) {
+    if (!value) {
+        return;
+    }
+
+    for (let color in colors) {
+        if (value.toLowerCase() === color.toLowerCase()) {
```

Comment:
Were we going to add a regex replace here to convert space to hyphens? Or am i thinking of somewhere else?